src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,13 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: 'Under construction',
```

Comment:
- [ ] In your code, you are using a string as the initial state for the categories slice, but the project requirements ask you to use an array of categories. To fix this, I kindly suggest that you need to change the `initialState` value to an empty array, like this:

```js
import { createSlice } from '@reduxjs/toolkit';

const categoriesSlice = createSlice({
 name: 'categories',
 initialState: [], // changed from 'Under construction'
 reducers: {
 checkStatus: () => 'Under construction',
 },
});

export const { checkStatus } = categoriesSlice.actions;

export default categoriesSlice.reducer;
```

This way, you can add categories to the categories state later on using the reducer functions. I hope this helps.😊